pypit/ardebug.py

```diff
@@ -17,3 +21,19 @@ def init():
                  trace_obj=False,
                  )
     return debug
+
+
+def show_image(args, **kwargs):
+    """ Wrapper to pypit_ginga
+    Parameters
+    ----------
+    args
+    kwargs
+
+    Returns
+    -------
+
+    """
+    return pyp_g.show_image(args, **kwargs)
+
+from pdb import *
```

Comment:
I see now what this does - and I agree that it is best. We should refactor this throughout (if xdb is no longer going to be used).